### ISSUE TITLE
[5.5.0] TargetFramework proposal

### ISSUE THREAD

**Issue by jan-maria** [fluentscheduler/FluentScheduler#279]
Hello,

Today I've noticed a new 5.5.0 version of FluentScheduler - great to see that the project evolves :)

I have a small proposal - wouldn't it be better to target netstandard2.0? Do we really need to have some features of the netstandard2.1? Unfortunately netstandard2.1 libraries won't work on .NET Framework

[https://docs.microsoft.com/pl-pl/dotnet/standard/net-standard](https://docs.microsoft.com/pl-pl/dotnet/standard/net-standard)
[https://github.com/fluentscheduler/FluentScheduler/blob/version-5/FluentScheduler/FluentScheduler.csproj](https://github.com/fluentscheduler/FluentScheduler/blob/version-5/FluentScheduler/FluentScheduler.csproj#L12)

Thank you,
Best regards,

**Comment by tallesl** [comment]
Unfortunately, it is needed [due to the `IAsyncJob` that was introduced](https://github.com/fluentscheduler/FluentScheduler/issues/117#issuecomment-667779897). If you downgrade the library to `netstandard 2.0` you get:

```
IAsyncJob.cs(19,19): error CS8370: Feature 'default interface implementation' is not available in C# 7.3. Please use language version 8.0 or greater.
IAsyncJob.cs(19,19): error CS8701: Target runtime doesn't support default interface implementation.
```

**Comment by tallesl** [comment]
5.5.1 should support .NET Standard 2.0 again.
